satpy/etc/composites/abi.yaml

```diff
@@ -215,3 +215,267 @@ composites:
     - 9.7
     - 10.3
     standard_name: airmass
+###
+  night_microphysics:
+    compositor: !!python/name:satpy.composites.Dust
+    prerequisites:
+    - name: C07
+    - name: C14
+    - name: C15
+    standard_name: night_microphysics
+
+  convection:
+    compositor: !!python/name:satpy.composites.Convection
+    prerequisites:
+    - name: C02
+    - name: C05
+    - name: C07
+    - name: C08
+    - name: C10
+    - name: C14
+    standard_name: convection
+
+## Cloud Phase Distinction or Day Cloud Phase (name from NASA SPORT) or Cloud Type RGB(name from EUMETSAT)
+  cloud_phase_distinction_raw:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C13
+    - name: C02
+    - name: C05
+    standard_name: cloud_phase_distinction
+
+  cloud_phase_distinction:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C13
+    - name: C02
+      modifiers: [sunz_corrected, rayleigh_corrected]
+    - name: C05
+      modifiers: [sunz_corrected]
+    standard_name: cloud_phase_distinction
+
+## Water Vapors RGB or Simple Water Vapor (name from NASA SPORT)
+  water_vapors1:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C13
+    - name: C08
+    - name: C10
+    standard_name: water_vapors1
+
+  mid_vapor:
+    compositor: !!python/name:satpy.composites.DifferenceCompositor
+    prerequisites:
+    - name: C10
+    - name: C08
+    standard_name: mid_vapor
+## Water Vapors RGB (other version) or Differential Water Vapor (name from NASA SPORT)
+  water_vapors2:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: mid_vapor
+    - name: C10
+    - name: C08
+    standard_name: water_vapors2
+
+## Day Cloud Phase RGB(name from EUMETSAT)
+  cloud_phase_raw:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C05
+    - name: C06
+    - name: C02
+    standard_name: natural
+
+  cloud_phase:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C05
+      modifiers: [sunz_corrected]
+    - name: C06
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected, rayleigh_corrected]
+    standard_name: natural
+##
+  ir_overview:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C05
+      modifiers: [co2_corrected]
```

Comment:
This (C05 with co2_corrected) gives me an empty dataset. Since this modifier wasn't created by you... @adybbroe @mraspaud any ideas?

Reply:
the co2 correction is specifically for SEVIRI
I don't think GOES would need it